File: Problem 1/sol.js

function plusMinus(arr) {
    const con = arr.length;
    let posi = 0, nega=0
    for(let i=0;i < con;i++){
        if(arr[i]<0)
            nega++;
        else if(arr[i]>0)
            posi++;
    }
    
    console.log((posi/con).toFixed(6))
    console.log((nega/con).toFixed(6))
    console.log(((con-posi-nega)/con).toFixed(6))
}

plusMinus([-4, 3, -9, 0, 4, 1]);

File: Problem 1/sol.cpp
void plusMinus(vector<int> arr) {
    int count = arr.size();
    int posi = 0;
    int nega = 0;
    
    for(int i=0;i<count;i++){
        if(arr[i]>0){
            posi++;
        }else if(arr[i]<0){
            nega++;
        }
    } 
    
    printf("%.4lf\n",static_cast<double>(posi)/count);
    printf("%.4lf\n",static_cast<double>(nega)/count);
    printf("%.4lf\n",static_cast<double>(count-posi-nega)/count);
    
}